cmd/repo-updater/shared/main.go

```diff
@@ -166,26 +165,26 @@ func Main(enterpriseInit EnterpriseInit) {
 		// We always want to listen on the Synced channel since external service syncing
 		// happens on both Cloud and non Cloud instances.
 		Synced:     make(chan repos.Diff),
-		Logger:     log15.Root(),
+		Logger:     log.Scoped("repo-updater", "Repo-updater tracks the state of repos."),
```

Comment:
```suggestion
		Logger:     logger,
```

We already have a logger instantiated